### PR TITLE
Flag aws_ssm as unstable rather than disabled

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

Flag aws_ssm as unstable rather than disabled

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

aws_ssm

##### ADDITIONAL INFORMATION

If possible #278 should trigger integration tests...

### CODE REVIEW

**Comment by tremble** [comment]
Tests blocked by https://github.com/mattclay/aws-terminator/pull/115

File: tests/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/templates/aws-env-vars.j2 [hunk 1]
```diff
@@ -1,4 +1,6 @@
 export AWS_ACCESS_KEY_ID={{aws_access_key}}
 export AWS_SECRET_ACCESS_KEY={{aws_secret_key}}
+{% if security_token is defined %}
```
Comment (jillr) [inline tests/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/templates/aws-env-vars.j2:3]: I'm getting token errors when testing with the updated CI policies.

```
botocore.exceptions.ClientError: An error occurred (UnrecognizedClientException) when calling the StartSession operation: The security token included in the request is invalid.
fatal: [i-0e7e9edaf5ff18ccb]: FAILED! => {
    "msg": "Unexpected failure during module execution.",
    "stdout": ""
}
```

Comment (tremble) [reply tests/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/templates/aws-env-vars.j2:3]: Wrong environment variable :)

**Comment by tremble** [comment]
@jillr I've been tinkering with this and I'm still hitting a few things:

1) If the connection is initiated too early then it doesn't connect properly, even with wait_for_connection.  I suspect this is a bug.
2) Windows takes a while to actually be ready.  By the time it finishes 1 run we're close to the 10 minute window and aws-terminator does its thing.
3) There is a bug in handling Unicode with Python 2.x - I wouldn't have caught this locally since I'm testing against devel /Python 3.6

The version I've just uploaded does work with Python 3.x when aws-terminator doesn't delete things.  You mentioned on IRC about increasing the aws-terminator delays.  Ultimately it's in our interest to have more comprehensive *stable* test suites, even if certain modules those tests are slow.  I've opened https://github.com/mattclay/aws-terminator/pull/117 to get the conversation going.

Since the vast bulk of our tests *do* clean-up, and the instances we use are pretty small the cost implications should be minimal.

**Comment by tremble** [comment]
@jillr I've been tinkering with this for a while.  Windows Instances take **way** too long to finish booting and get ready.  I've disabled the Windows pieces so we're at least testing this on Linux.  How do you feel about enabling these tests with Windows disabled?

**Review by jillr — APPROVED** [review]

**Pull request merged**